### DISCUSSION TITLE
How to get data sqlite from ionic to native android

### DISCUSSION THREAD

**Discussion by aditiam** [capacitor-community/sqlite#416] [Q&A] [answered]
I try to achieve when user login and the data save into sqlite from ionic, i want to call it from android project using dependency "SQLiteOpenHelper" when i using it and initialize database name it's look like overwrite the database when i call it from ionic project, any suggestion and advice?

**Comment by jepiqueau** [comment, marked as answer]
Look at all the app links in the readme of the @capacitor-community/sqlite. You will understand how it is setup and how it works
